src/task/FinishGatheringICECandidatesTask.ts

```diff
@@ -76,9 +76,13 @@ export default class FinishGatheringICECandidatesTask extends BaseTask {
       );
     }
 
-    // To bypass waiting for events, it is required that "icegatheringstate" to be complete and sdp to have candidate
+    /*
+     * To bypass waiting for events, it is required that "icegatheringstate" to be complete and sdp to have candidate
+     * For Firefox, it takes long for iceGatheringState === 'complete'
```

Comment:
I think we need a bit more explanation, e.g. why only Firefox but not for other supported browsers?